packages/tools/viewer/src/viewer.ts

```diff
@@ -134,7 +137,30 @@ function throwIfAborted(...abortSignals: (Nullable<AbortSignal> | undefined)[]):
     }
 }
 
-function createSkybox(scene: Scene, camera: Camera, reflectionTexture: CubeTexture, blur: number): Mesh {
+async function createCubeTexture(url: string, scene: Scene, extension?: string) {
+    extension = extension ?? GetExtensionFromUrl(url);
+    const instantiateTexture = await (async () => {
+        if (extension === ".hdr") {
+            // eslint-disable-next-line @typescript-eslint/naming-convention
+            const { HDRCubeTexture } = await import("core/Materials/Textures/hdrCubeTexture");
+            return () => new HDRCubeTexture(url, scene, 128, false, true, false, true);

```

Comment:
Can we use new `HDRCubeTexture(url, scene, 256, false, true, false, true, undefined, undefined, undefined, true, true);`

Reply:
Yes, if that's the right configuration I will change it!